docs/reference/mapping/types/runtime.asciidoc

```diff
@@ -0,0 +1,90 @@
+[[runtime]]
+=== Runtime
+
+////
+[source,console]
+----
+PUT /my-index
+----
+// TESTSETUP
+////
+
+Typically, you must index fields to {es} before they can be retrieved,
+aggregated, or searched. With runtime fields, you can explicitly define a field
+in the mapping and access it at runtime without indexing your data. This
+flexibility allows you to more quickly ingest raw data into the Elastic Stack
+and immediately access it. By dynamically evaluating runtime fields at query
+time, you can choose which fields to index and optimize disk space.
+
+Runtime fields are not indexed, but are accessible from the search API like any
+other field that has <<doc-values,doc_values>> and is searchable. You can
+retrieve and query these fields, as well as aggregate and
+sort on them.
+
+NOTE: Runtime fields can be incur performance costs at search time, depending
+on the <<runtime-params-runtime-type,`runtime_type`>>.
```

Comment:
Two concerns here:

1. This shouldn't be a rehash of the field type description. That's just a quick, condensed preview. On this page, you should take some space to better lay out the problem, give more specific use cases, and cover considerations.
2. The info hierarchy is a little off. Here's how I'd tackle it in order:

- What problems do runtime fields solve? How?
- What are the tradeoffs of using runtime fields? E.g., lower storage costs, faster indexing vs. slower search
- When are runtime fields commonly used? What use cases? E.g., time series data, logging, etc.
- What are some interesting ways I could use it? Examples?
- What are some considerations before I use it? Are there fields I should _not_ use as runtime fields?